internal/gitlab/gitlab.go

```diff
@@ -41,6 +45,10 @@ func User() string {
 	return user
 }
 
+func Client() *gitlab.Client {
```

Comment:
Let's go ahead and not poke a hole in this package. Any functionality needed from the client should be exposed through this package. Even if said functions, are silly wrappers, still best to stay consistent